src/modules/admin/const/Requests.const.tsx

```diff
@@ -1,32 +1,9 @@
-import { TabProps } from '@meemoo/react-components';
 import { Column } from 'react-table';
 
 import { CopyButton, Icon, RequestStatusBadge } from '@shared/components';
 import { i18n } from '@shared/helpers/i18n';
-import { Visit, VisitRow, VisitStatus } from '@shared/types';
+import { Visit, VisitRow } from '@shared/types';
 import { asDate, formatDistanceToday, formatMediumDateWithTime } from '@shared/utils';
-import { RequestStatusAll } from '@visits/types';
-
-export const requestStatusFilters = (): TabProps[] => {
```

Comment:
Used in https://github.com/viaacode/hetarchief-client/blob/bugfix/remove-unused-code/src/modules/visits/components/VisitRequestsOverview/VisitRequestsOverview.tsx#L92